base/loading.jl

```diff
@@ -341,7 +337,7 @@ during package development as code is modified.
 function reload(name::AbstractString)
     if isfile(name) || contains(name,Filesystem.path_separator)
         # for reload("path/file.jl") just ask for include instead
-        error("use `include` instead of `reload` to load source files")
+        error("Use `include` instead of `reload` to load source files.")
```

Comment:
It hurts if half the error messages are capitalized and half are not. Seems to make more sense to me to have a separate PR that changes all the error messages to be capitalized if that is the way we want them to be. There have been PRs that have changed error messages from capitalized to non capitalized so something should be decided.